# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/test/unit/head_channels.js
# Commit: db100843b0f0
# Full Hash: db100843b0f00b89944d4dcbcb2439a48b33eaab
# Author: Valentin Gosu <valentin.gosu@gmail.com>
# Date: 2024-12-03 21:40:41
# Regressor Bug: 1929156
# File Overlap Count: 1
# Description:
#   Bug 1929156 - Check negotiated ALPN matches altSvc protocol r=necko-reviewers,kershaw
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D228217
# ==============================================================================

diff -r 16c36964a84b -r db100843b0f0 netwerk/test/unit/head_channels.js
--- a/netwerk/test/unit/head_channels.js	Tue Dec 03 13:50:03 2024 +0000
+++ b/netwerk/test/unit/head_channels.js	Tue Dec 03 14:06:27 2024 +0000
@@ -377,6 +377,7 @@
   Services.env.set("LD_LIBRARY_PATH", greBinDir.path + ":/data/local/xpcb");
   Services.env.set("MOZ_TLS_SERVER_DEBUG_LEVEL", "3");
   Services.env.set("MOZ_TLS_SERVER_CALLBACK_PORT", CALLBACK_PORT);
+  Services.env.set("MOZ_TLS_ECH_ALPN_FLAG", "1");
 
   let httpServer = new HttpServer();
   let serverReady = new Promise(resolve => {
